src/js/wrap.cpp

```diff
@@ -201,41 +203,41 @@ namespace js
     JS_FreeCString(ctx, property_name_c);
     LOG_TRACE_FMT("Looking for kv map '{}'", property_name);
 
-    const auto [security_domain, access_category] =
-      kv::parse_map_name(property_name);
+    // const auto [security_domain, access_category] =
+    //   kv::parse_map_name(property_name);
 
     auto read_only = false;
-    switch (access_category)
-    {
-      case kv::AccessCategory::INTERNAL:
-      {
-        if (security_domain == kv::SecurityDomain::PUBLIC)
-        {
-          read_only = true;
-        }
-        else
-        {
-          throw std::runtime_error(fmt::format(
-            "JS application cannot access private internal CCF table '{}'",
-            property_name));
-        }
-        break;
-      }
-      case kv::AccessCategory::GOVERNANCE:
-      {
-        read_only = true;
-        break;
-      }
-      case kv::AccessCategory::APPLICATION:
-      {
-        break;
-      }
-      default:
-      {
-        throw std::logic_error(fmt::format(
-          "Unhandled AccessCategory for table '{}'", property_name));
-      }
-    }
+    // switch (access_category)
```

Comment:
As discussed, commented out for now so that we can mutate the KV from JS actions.